app/submitter/submitter.py

```diff
@@ -141,3 +143,31 @@ def send_message(self, message, tx_id, questionnaire_id, case_id=None):
         finally:
             if connection:
                 self._disconnect(connection)
+
+
+class GCSFeedbackSubmitter:
+    def __init__(self, bucket_name):
+        client = storage.Client()
+        self.bucket = client.get_bucket(bucket_name)
+
+    def upload(self, metadata, payload):
+        blob = self.bucket.blob(str(uuid4()))
+        blob.metadata = metadata
+        blob.upload_from_string(
+            str({**payload, **metadata}).encode("utf8"), content_type="application/json"
+        )
```

Comment:
Since this is explicitly needed as JSON this should be:
```suggestion
        blob.upload_from_string(
            data=json.dumps(payload), content_type="application/json"
        )
```

Reply:
Content type is set, so is this needed?

Reply:
Yes, content type is used by the browser only, atm, this is still a string so json serialistion is needed.